grails-app/services/com/unifina/service/CanvasService.groovy

```diff
@@ -46,6 +46,9 @@ class CanvasService {
 
 	@CompileStatic
 	void updateExisting(Canvas canvas, SaveCanvasCommand command, SecUser user, boolean resetUi = false) {
+		if (command.name == null || command.name.trim() == "") {
```

Comment:
Does it work if the default `name` is in `SaveCanvasCommand`, just like for domain objects? 
Could the code be more DRY, so that the default name is not repeated in multiple places? Something like:

In `SaveCanvasCommand`:

`String name = Canvas.DEFAULT_NAME`

In `Canvas`:

`public static String DEFAULT_NAME = "Untitled Canvas"`
...
`String name = Canvas.DEFAULT_NAME`

Reply:
👍Added DEFAULT_NAME